### DISCUSSION TITLE
Меню

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/gothic2_steam_fix#19] [General]
первый пост. можно тут что то делать типа списка

- [ ] вкл / выкл субтитры
- [ ] управление Готика 1
- [ ] вкл / выкл что то там еще

**Comment by Kor-Angar** [comment]
В меню раньше было включение / отключение субтитров. у меня что то обновилось, сбросились настройки и пропали субтитры. возможно это произошло после того как слетел юнион и готика начало совершать действия какие совершала в 2002 году :-)
![2021-11-06_15-30-01](https://user-images.githubusercontent.com/49514764/140613623-5e0cbc7b-0e2a-4c45-9ccd-2e847d67de9e.png)

**Reply by dosinabox** [reply]
А что есть в меню сейчас? Может настройка переехала в другой раздел?

**Reply by Kor-Angar** [reply]
Неоф. обновление | G2 NotR DE 2.6
-- | --
![20211106182204_1](https://user-images.githubusercontent.com/49514764/140614814-2391ed2d-d8b5-467b-8805-8e737aae6c38.jpg) | ![20211106182134_1](https://user-images.githubusercontent.com/49514764/140614815-ce53427f-ecac-41f3-83d7-acafb1bad4b0.jpg)

**Reply by Kor-Angar** [reply]
вообщем эти настройки были прописаны в готики.ини который берётся из ресурсов игры(?), если слетит Системпак. 
Просто имей ввиду что такое может произойти, ну и конечно же когда то у кого то произойдёт.

текст | картинка
-- | --
Дефаутные настройки у Пираней стоят без субтитров, во всех мануалах написано что включить их можно в меню, а в твоём меню (и иностранных версиях наверное тоже?) этот переключатель пропал. | ![image](https://user-images.githubusercontent.com/49514764/140616702-6f944d06-9e34-4654-a285-ec1a17166e3f.png)[Handbuch.pdf](https://github.com/dosinabox/g2_steam_fix/files/7490849/Handbuch.pdf)

**Reply by dosinabox** [reply]
В обновлении эта настройка переехала в раздел настроек звука:
![Screenshot_1](https://user-images.githubusercontent.com/39790055/140625451-a757a9af-5f46-4518-a8c6-fea2f2cc2e78.jpg)

**Reply by Kor-Angar** [reply]
неочевидно что настройки с текстом, надо искать в настройках звука. вроде как звук я воспринимаю ушами, а субтитры(видео) глазами. В штатах тебя могли бы глухие засудить за их дискриминацию  ;-)

**Comment by Kor-Angar** [comment]
"управление Готика 1" и настройки мышки можно перенести в меню "**управление**"
![image](https://user-images.githubusercontent.com/49514764/140628308-f97979a7-23dd-4d84-a5a9-1c732378b879.png)

![20211107035248_1](https://user-images.githubusercontent.com/49514764/140628293-f6254656-8d50-488f-a8e1-5db548975759.jpg)
е
